src/components/Footer/Footer.module.css

```diff
@@ -1,94 +1,94 @@
 .Footer {
```

Comment:
I think by mistake you have also pushed the changes for footer in this PR.

Reply:
Thx brother! 😄😄

Reply:
rectified